OpenSearch - Missing object storage in configurer network policy

**Describe the bug**
During a clean install of the service cluster apps v0.28.0, the OpenSearch configurer was unable to access the s3 storage. After we removed all network policies in the opensearch namespace it finally succeeded.

**To Reproduce**
Steps to reproduce the behavior:
1. Have a service cluster without any apps.
2. Run `bin/ck8s apply sc`
3. It should get stuck at the opensearch-configurer

**Expected behavior**
The configurer should be able to access s3 storage and succeed. 

**Version (add all relevant versions):**
 - Compliant kubernetes apps version v0.28.0

**Additional context**
Looking into the policies, it seems as if the [policy targeting the configurer](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/networkpolicy-service/templates/opensearch/configurer.yaml) does not have the object storage config. 

It seems as if object storage is only configured in [master.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/networkpolicy-service/templates/opensearch/master.yaml) which does not select the configurer.

Hi, as I looked into the logs now it seems as if both a master, a data and the configurer logged the same type of error:

`Caused by: com.amazonaws.SdkClientException: Unable to execute HTTP request: Connect to [ip-hidden] failed: Connect timed out`

We have had issues with s3 access from opensearch for a few days but was successful today which seemed to be when we removed the netpols, the configurer never managed to complete before.

--- a/WIP-CHANGELOG.md
+++ b/WIP-CHANGELOG.md
@@ -11,6 +11,7 @@
 
 - The update-ips script can now fetch Calico Wireguard IPs
 - The test scripts can now always access the kubeconfig
+- The OpenSearch network policies now properly work with dedicated nodes and shapshots enabled
 
 ### Updated
 

--- a/helmfile/charts/networkpolicy-service/templates/opensearch/client.yaml
+++ b/helmfile/charts/networkpolicy-service/templates/opensearch/client.yaml
@@ -57,6 +57,16 @@ spec:
       ports:
         - port: 9300
     - to:
+        {{- range $ip := .Values.global.objectStorage.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      ports:
+        {{- range $port := .Values.global.objectStorage.ports }}
+        - protocol: TCP
+          port: {{ $port }}
+        {{- end }}
+    - to:
         {{- range $ip := .Values.opensearch.plugins.ips }}
         - ipBlock:
             cidr: {{ $ip }}

--- a/helmfile/charts/networkpolicy-service/templates/opensearch/data.yaml
+++ b/helmfile/charts/networkpolicy-service/templates/opensearch/data.yaml
@@ -39,6 +39,16 @@ spec:
       ports:
         - port: 9300
     - to:
+        {{- range $ip := .Values.global.objectStorage.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      ports:
+        {{- range $port := .Values.global.objectStorage.ports }}
+        - protocol: TCP
+          port: {{ $port }}
+        {{- end }}
+    - to:
         {{- range $ip := .Values.opensearch.plugins.ips }}
         - ipBlock:
             cidr: {{ $ip }}

